src/Keboola/CsvTable/Table.php

```diff
@@ -75,11 +75,20 @@ public function getAttributes(): array
 
 	public function setIncremental(bool $incremental): void
 	{
-		$this->incremental = (bool) $incremental;
+		$this->incremental = $incremental;
 	}
 
+	public function isIncrementalSet(): bool
```

Comment:
Tato nova metoda sluzi na to, aby sa dalo skontrolovat ci bola hodnota nastavena. Teda `getIncremental` nespadne na null hodnote + moze stale vraciat bool.